vector-app/build.gradle

```diff
@@ -340,16 +340,48 @@ android {
                 "-opt-in=kotlinx.coroutines.ExperimentalCoroutinesApi",
         ]
     }
+
+    buildFeatures {
+        viewBinding true
+    }
```

Comment:
Sorry, I had removed this in #7014 because it was not used. There are maybe other changes to revert on my PR.

Reply:
I'll smoke test the different builds :crossed_fingers:  this was added as it's now used by the debug fragments

Reply:
can confirm fdroid, nightly, debug and release builds can log in and send a message